examples/inference-deployments/mpt/mpt_7b_ft_handler.py

```diff
@@ -219,8 +216,9 @@ def _parse_inputs(
 
         return generate_inputs, generate_kwargs
 
-    def predict(self, input_dicts: List[Dict[str, Any]]) -> List[str]:
-        generate_inputs, generate_kwargs = self._parse_inputs(input_dicts)
+    def predict(self, model_requests: List[Dict[str, Any]]) -> List[str]:
```

Comment:
the new request structure matches the [HF schema now](https://huggingface.github.io/text-generation-inference/#/Text%20Generation%20Inference/generate), right? worth documenting?

Reply:
This was completely by accident! We had no idea. I can see a subtle difference where we must take in a list of inputs while HF takes in just one input as a string. This is because we need to be compatible for other types of deployments not just inference for NLP.